### ISSUE TITLE
unable to spend sats

### ISSUE THREAD

**Issue by cast4way** [MutinyWallet/mutiny-web#1041]
Today I added some sats to my wallet, I wanted to spend them from my selfcustody ln wallet, but they automatically where put into my esats wallet from fedi. I tried to swap the sats into self custody but now I cannot swap or spend any of the two balances.
Is there a 24h reset or something on pending transactions? Because from what I understand, one of my transactions is stuck and this causes all to be stuck.
[Uploading mutiny-logs.txt…]()

Help is much appreciated. Cheers

**Comment by cast4way** [comment]
[mutiny-logs.txt](https://github.com/MutinyWallet/mutiny-web/files/14959759/mutiny-logs.txt)
I hope it works this time.

**Comment by laantungir** [comment]
I am having a similar issue.

[mutiny-logs.txt](https://github.com/MutinyWallet/mutiny-web/files/14959898/mutiny-logs.txt)

**Comment by TonyGiorgio** [comment]
which federation are you both on? they could be experiencing issues. I see payment failures coming from them, which we don't have any control over.

**Comment by cast4way** [comment]
Bitcoin Atlantis

**Comment by TonyGiorgio** [comment]
Where are you sending to? Try sending to different places, smaller amounts, etc.

**Comment by cast4way** [comment]
[mutiny-logs-1.txt](https://github.com/MutinyWallet/mutiny-web/files/14960813/mutiny-logs-1.txt)
500 sats swap fails, 200 sats to other wallet fails too.

**Comment by TonyGiorgio** [comment]
You already have a lightning channel that you're trying to swap into? I see lots of LSP disconnection issues, can you see if your channel shows as online in the settings, and does it stay stable for 30+ seconds?

What other wallets?

**Comment by cast4way** [comment]
I tested minibits and phoenix. Yes I have a LN channel with mutiny

**Comment by TonyGiorgio** [comment]
> Yes I have a LN channel with mutiny

can you see if your channel shows as online in the settings, and does it stay stable for 30+ seconds?

**Comment by laantungir** [comment]
I am also on Atlantis federation. I have transfered over eCash, but have not opened a LN channel.

**Comment by TonyGiorgio** [comment]
> I am also on Atlantis federation. I have transfered over eCash, but have not opened a LN channel.

wasn't able to get anything from your logs. Could you try to make the a payment again and after it fails, upload the logs again?

**Comment by laantungir** [comment]
I just tried sending 15 sats from laantungir@mutiny.plus to tony@mutiny.plus and it said payment pending.
[mutiny-logs(1).txt](https://github.com/MutinyWallet/mutiny-web/files/14960958/mutiny-logs.1.txt)

**Comment by TonyGiorgio** [comment]
@cast4way are you also using ligtning addresses with mutiny.plus?

**Comment by laantungir** [comment]
So Tony then you think it is a Bitcoin Atlantis issue? I think I tried other federations, but I can try again. Is there one you know that works that I should try? I will attempt to send to you again.

**Comment by TonyGiorgio** [comment]
I'm switching between both federations and paying multiple wallets and it's working on my device. I don't think there's something wrong with the Atlantis federation itself, but perhaps something wrong between both of your devices talking to the federation. I've reached out to the fedimint team to understand how to troubleshoot more. 

You can switch federations, but the lightning address won't switch yet, that's coming with the next release which we are submitting today and should be out early next week.

**Comment by TonyGiorgio** [comment]
Just to confirm, the balance is still updating correctly, right? As in, it goes down for the payment but goes back to what it was when it failed?

**Comment by laantungir** [comment]
Yes, my balance drops when I attempt to send, but then after it fails, the balance returns back to pre-send level.

**Comment by laantungir** [comment]
I'll wait to see if something comes from the fediment issue before I try anything else. Thanks.

**Comment by cast4way** [comment]
I do not use mutiny +.

I think I caused the issue by paying a hodl invoice with the fedi esats. But I was not aware about it, thought I was using my self custody sats. At least that was the first failing payment.

Any idea why the app is lagging? Pressing any puttom causes a minimum 10sec delay, which is not normal. 

Been using mutiny for month without problems

**Comment by cast4way** [comment]
> > Yes I have a LN channel with mutiny
> 
> can you see if your channel shows as online in the settings, and does it stay stable for 30+ seconds?

My channel is offline it seems. How do I fix that.

**Comment by TonyGiorgio** [comment]
So either of you use mutiny from multiple devices?

**Comment by cast4way** [comment]
No, I use the android app v0.6.3.

**Comment by TonyGiorgio** [comment]
I'm investigating the reason for slowness now. It's related to your fail payments but I don't believe @laantungir is showing that symptom. Could be for failed swaps specifically. 

One thing that might work for you is to restore from seed words again. This will wipe all metadata but keep your balances, federations, funds, etc. I don't think this will fix your payment problem, but it might clear up the reason it's lagging.

**Comment by laantungir** [comment]
I am using Mutiny in a browser. I also have it installed on Android (Graphine OS), but I have been aware of having it open on multiple devices, and have made sure it is closed out on Android.

**Comment by laantungir** [comment]
I restored from seed words. Doesn't seem to change anything.

**Comment by cast4way** [comment]
> I'm investigating the reason for slowness now. It's related to your fail payments but I don't believe @laantungir is showing that symptom. Could be for failed swaps specifically.
> 
> One thing that might work for you is to restore from seed words again. This will wipe all metadata but keep your balances, federations, funds, etc. I don't think this will fix your payment problem, but it might clear up the reason it's lagging.

Restored and lagging is fixed 🙏.
Payment problem remains.

**Comment by cast4way** [comment]
Payment issue still remains:
[mutiny-logs-2.txt](https://github.com/MutinyWallet/mutiny-web/files/14968219/mutiny-logs-2.txt)
Is there a debug I can try from the admin page?
Seems like my lightning channel goes offline...but even when its online I cannot move funds.

**Comment by TonyGiorgio** [comment]
There's still multiple people with issues with the Atlantis federation and I haven't heard back from them on what the issues are. To confirm, the people here currently having issues, were you ever able to spend out of that federation? 

Has this occurred on any of the other federations?

**Comment by newtonick** [comment]
I'm having a similar issue where I can't spend or sweep from the Bitcoin Atlantis federation. I haven't tried another federation because I wasn't sure if I would lose the funds completely and/or if my mutiny.plus lnaddress would transfer to the new federation. I figured I would wait before trying another federation.

I am able to spend and recieve funds from my lightning balance (1 channel currently). As long as I'm paying an invoice larger than the federation esat balance, then the payment works. I am not using the LSP. I opened a channel from my own node. I have deleted the LSP on the settings -> server page. 

Here are my logs: [mutiny-logs.txt](https://github.com/MutinyWallet/mutiny-web/files/14985430/mutiny-logs.txt)

6 days ago I was able to send and sweep funds from the federation. Then it stopped working. I became a mutiny plus subscriber and signed up for a mutiny.plus lnaddress around this same time. I'm not sure if spending from the federation ever worked after I enabled mutiny plus. I'm not sure if I can even receive funds into the federation at this point.

Here as screen recordings of swap and spend failures. They both just "fail" to the "pending payment" screen. The logs seems to indicate a some kind of GatewayInternalError.

Swap attempt:
https://github.com/MutinyWallet/mutiny-web/assets/127377/01c551df-cae0-42aa-96af-1b9e1bdf4a28

Send attempt:
https://github.com/MutinyWallet/mutiny-web/assets/127377/ef0dd2f9-f1fa-4afc-ab3a-e60149973989

Hopefully this information helps identify the issue. Appreciate the work you're doing!

**Comment by newtonick** [comment]
Quick update: I was able to spend out of Mutiny from my Bitcoin Atlantis federation balance to the Fedi Bravo app (where I believe I am using the same federation). This means I was able to move all of my federated funds out of Mutiny and leave the federation.

I then joined a different federation and was able to receive (via my mutiny plus lnaddress) and sweep/spend from the federation balance without any issue.

**Comment by TonyGiorgio** [comment]
This is very helpful, thank you @newtonick. Will be troublshooting this more with the fedimint team.

Can you spend out of your fedi bravo app now that you moved your funds there? It does make sense that you were able to spend within the federation to a different client, since that bypasses the lightning gateway for the federation, which seems to be the one reporting the issue.

**Comment by newtonick** [comment]
> Can you spend out of your fedi bravo app now that you moved your funds there? It does make sense that you were able to spend within the federation to a different client, since that bypasses the lightning gateway for the federation, which seems to be the one reporting the issue.

I was able to spend from Fedi Bravo.

**Comment by TonyGiorgio** [comment]
It does seem like in most of these cases so far, the common denominator was either a hodl invoice or a swap to lightning, which in the case of our LSP, uses a hodl invoice. 

I'll do a deeper dive tomorrow into prolonged / failed payments with fedimint and see if I can reproduce this. Thanks for all the information provided so far. It does appear though that funds are recoverable by transfering to fedi bravo but I'll try to figure out a fix soon.

**Comment by TonyGiorgio** [comment]
I think we have a fix, will try to get it deployed asap.

**Comment by cast4way** [comment]
Thanks @newtonick sending to Fedi Bravo on the same federation and spending from there worked for me to unstuck my fedi funds.
